### PR TITLE
Updated cookie authorisation to use standard gov uk style banner

### PR DESCRIPTION
**Pull request by rmbielby**
## Pull request overview

Switching to the shiny gov style cookie banner following the work done on the KS4 dashboard.

image

## Pull request checklist

Please check if your PR fulfils the following:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Tests have been run locally and are passing (`run_tests_locally()`)
- [ ] Code is styled according to tidyverse styling (checked locally with `tidy_code()`)

### CODE REVIEW

**Review by cjrace — CHANGES_REQUESTED** [review]

File: google-analytics.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
 </script>
 
 <!-- Global site tag (gtag.js) - Google Analytics -->
-  <script async src="https://www.googletagmanager.com/gtag/js?id=G-Z967JJVQQX"></script>
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-XXXXXXXXXX"></script>
```
[the remark this answers is not in the document]
Comment (rmbielby) [reply google-analytics.html:15]: Ah, that's from one of Jen's commits from a little while back, we must not have added her into the commit hook that resets it.

[the remark this answers is not in the document]
Comment (rmbielby) [reply google-analytics.html]: Ah, Jen has been added, but it looks like the wrong credentials. Updated now.

File: google-analytics.html [hunk 2]
```diff
@@ -22,7 +22,7 @@
 /*
 The tracking number below MUST be replaced with a unique number, contact the statistics development team to set this up. 
 */
-  gtag('config', 'G-Z967JJVQQX');
+  gtag('config', 'G-XXXXXXXXXX');
```
[the remark this answers is not in the document]
Comment (rmbielby) [reply google-analytics.html]: Fixed as part of above

File: renv/activate.R [hunk 3]
```diff
@@ -2,7 +2,7 @@
 local({
 
   # the requested version of renv
-  version <- "1.0.3"
+  version <- "1.0.2"
```
[the remark this answers is not in the document]
Comment (rmbielby) [reply renv/activate.R]: Nope, it's definitely 1.0.3 now

**Review by cjrace — COMMENTED** [review]
Not sure if this was pre-existing but the reset button doesn't seem to let you genuinely reset in all cases?

1.) Start app locally and open in browser -> Accept. Then clicking 'reset' removes the line of explanatory text but still seems to be printing tags in the console?

2.) Start app locally and open in browser -> Reject. Then clicking 'reset' removes the line of explanatory text and prints tags to console fine, but you can keep pressing it and I'm not sure what that does? Should it be more of an on / off toggle? 

Basically, I'm confident the initial bit works but if you do want to change them later it doesn't feel obvious what is happening and if you're actually able to? 

Could do similar to what we have on EES as an example - https://explore-education-statistics.service.gov.uk/cookies?

One other issue, is that the line of text explaining how we use the cookies doesn't appear if you start it up and then click view cookies straightaway (as some users may, to inform their decision on whether to accept or reject?).

I can't see obvious changes causing these, so if they were pre-existing then happy to raise these as separate issues to address in a different PR

**Comment by rmbielby** [comment]
> Not sure if this was pre-existing but the reset button doesn't seem to let you genuinely reset in all cases?
> 
> 1.) Start app locally and open in browser -> Accept. Then clicking 'reset' removes the line of explanatory text but still seems to be printing tags in the console?
> 
> 2.) Start app locally and open in browser -> Reject. Then clicking 'reset' removes the line of explanatory text and prints tags to console fine, but you can keep pressing it and I'm not sure what that does? Should it be more of an on / off toggle?
> 
> Basically, I'm confident the initial bit works but if you do want to change them later it doesn't feel obvious what is happening and if you're actually able to?
> 
> Could do similar to what we have on EES as an example - https://explore-education-statistics.service.gov.uk/cookies?
> 
> One other issue, is that the line of text explaining how we use the cookies doesn't appear if you start it up and then click view cookies straightaway (as some users may, to inform their decision on whether to accept or reject?).
> 
> I can't see obvious changes causing these, so if they were pre-existing then happy to raise these as separate issues to address in a different PR

I think I might just take the "pre-existing" get out clause you've put on these ones. And at least some of the above now comes under dfeshiny rather than the template, so if you're ok with it being deferred to a separate PR, then I think that's my preference.

**Comment by cjrace** [comment]
No problem, let's do that and get this merged in

**Review by cjrace — APPROVED** [review]

**Pull request merged**